passenger-service/src/main/java/com/modsen/passengerservice/dto/exception/ValidationErrorResponse.java

```diff
@@ -0,0 +1,17 @@
+package com.modsen.passengerservice.dto.exception;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Map;
+
+@Setter
+@Getter
+public class ValidationErrorResponse extends ExceptionResponse {
+
+    private Map<String, String> errors;
+
+    public ValidationErrorResponse(Integer status, String message) {
+        super(status, message);
+    }
+}
```

Comment:
Над ним можно добавить ломбоковскую анатацию `@Builder`, и тогда не придётся одельно инициадизировать статус код с месседжем и errors
Можно вообще сделать так:
```suggestion
@Builder
public record ValidationErrorResponse(
        @NonNull
        Integer status,
        @NonNull
        String message,
        @NonNull
        Map<String, String> errors
) {
}
```